cpg_workflows/stages/large_cohort.py

```diff
@@ -125,45 +131,122 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
         return self.make_outputs(cohort, self.expected_outputs(cohort), [j])
 
 
-@stage(required_stages=[SampleQC, DenseSubset])
-class Relatedness(CohortStage):
+@stage(required_stages=DenseSubset, analysis_keys=['relatedness_ht'], analysis_type='mt')  # is MT an analysis type?
+class RelatednessPCRelate(CohortStage):
+    """
+    This is the first step of the Relatedness Stage, without Dataproc
+    """
+
     def expected_outputs(self, cohort: Cohort) -> dict[str, Path]:
-        if relatedness_version := config_retrieve(['large_cohort', 'output_versions', 'relatedness'], default=None):
-            relatedness_version = slugify(relatedness_version)
+        return {
+            'relatedness_ht': (
+                cohort.analysis_dataset.prefix() / get_workflow().name / relatedness_version() / 'relatedness.ht'
+            ),
+        }
 
-        relatedness_version = relatedness_version or get_workflow().output_version
-        relatedness_path = (
-            cohort.analysis_dataset.prefix() / get_workflow().name / relatedness_version / 'relatedness.ht'
-        )
-        relatedness_to_drop_path = (
-            cohort.analysis_dataset.prefix() / get_workflow().name / relatedness_version / 'relateds_to_drop.ht'
+    def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput:
+
+        outputs = self.expected_outputs(cohort)
+        dense_mt_path = str(inputs.as_path(cohort, DenseSubset))
+        dense_mt_name = dense_mt_path.split('/')[-1]
+
+        # estimate required storage - we create a new HT, so provision double the storage
+        required_storage = (
+            tshirt_mt_sizing(
+                sequencing_type=config_retrieve(['workflow', 'sequencing_type']),
+                cohort_size=len(cohort.get_sequencing_group_ids()),
+            )
+            * 2
```

Comment:
Just tried kicking off a test batch which failed because `tshirt_mt_sizing` returns a string, so `50Gi` * 2 becomes `50Gi50Gi` 😅
```suggestion
            t_shirt_size_value = tshirt_mt_sizing(
            sequencing_type=config_retrieve(['workflow', 'sequencing_type']),
            cohort_size=len(cohort.get_sequencing_group_ids()),
        ).split('Gi')[0]
        required_storage_value = int(t_shirt_size_value) * 2
        required_storage = f'{required_storage_value}Gi'
```